src/mqttcm_connect.c

```diff
@@ -143,6 +159,14 @@ int mqttCMRbusInit(char *pComponentName)
 		return 0;
 	}
 	MqttCMInfo("mqttCMRbusInit is success. ret is %d\n", ret);
+
+
+	err = pthread_create(&disconnect_tid, NULL, disconnect_thread,NULL);
```

Comment:
Disconnect thread cannot be added in rbus init func.

Reply:
created function for thread